pkg/psalabelsyncer/podsecurity_label_sync_controller_test.go

```diff
@@ -40,6 +40,49 @@ func testNamespaces() []*corev1.Namespace {
 		{ObjectMeta: metav1.ObjectMeta{Name: "controlled-namespace-previous-enforce-version-different-owner", Annotations: map[string]string{securityv1.UIDRangeAnnotation: "1000/1052"}, Labels: map[string]string{psapi.EnforceVersionLabel: "bogus version value"}, ManagedFields: managedLabelsFields("someone-else", psapi.EnforceVersionLabel)}},
 		{ObjectMeta: metav1.ObjectMeta{Name: "controlled-namespace-previous-warn-labels", Annotations: map[string]string{securityv1.UIDRangeAnnotation: "1000/1052"}, Labels: map[string]string{psapi.WarnLevelLabel: "bogus value", psapi.WarnVersionLabel: "bogus version value"}, ManagedFields: managedLabelsFields("cluster-policy-controller", psapi.WarnLevelLabel, psapi.WarnVersionLabel)}},
 		{ObjectMeta: metav1.ObjectMeta{Name: "non-controlled-namespace", Labels: map[string]string{"security.openshift.io/scc.podSecurityLabelSync": "false"}, Annotations: map[string]string{securityv1.UIDRangeAnnotation: "1000/1052"}}},
+		{ObjectMeta: metav1.ObjectMeta{
+			Name:        "controlled-namespace-different-owner-and-sync-label-true",
+			Annotations: map[string]string{securityv1.UIDRangeAnnotation: "1000/1052"},
+			Labels: map[string]string{
+				psapi.EnforceLevelLabel:                          "bogus enforce level value",
+				psapi.EnforceVersionLabel:                        "bogus enforce version value",
+				"security.openshift.io/scc.podSecurityLabelSync": "true",
+			},
+			ManagedFields: managedLabelsFields("someone-else",
+				psapi.EnforceLevelLabel,
+				psapi.EnforceVersionLabel,
+			),
+		}},
+		{ObjectMeta: metav1.ObjectMeta{
+			Name:        "non-controlled-namespace-different-owner-and-sync-label-false",
```

Comment:
What is the meaning of `non-controlled` and `controlled`? The only difference between this and the namespace above is the label sync value, which is mentioned explicitly at the end of the namespace's name.

Reply:
as far as i see this denotes what namespaces are "controlled" or "not controlled" by the policy controller.